includes/Admin/Orders.php

```diff
@@ -374,19 +375,9 @@ public function render_refund_reason_field( $select_id = 'wc_facebook_refund_rea
 	 * @since 2.1.0-dev.1
 	 *
 	 * @param int $refund_id refund ID
-	 *
-	 * @throws Framework\SV_WC_Plugin_Exception
 	 */
 	public function handle_refund( $refund_id ) {
 
-		$order_refund = wc_get_order( $refund_id );
-
-		if ( $order_refund instanceof \WC_Order_Refund ) {
-
-			$reason_code = isset( $_POST['wc_facebook_refund_reason'] ) ? $_POST['wc_facebook_refund_reason'] : null;
-
-			facebook_for_woocommerce()->get_commerce_handler()->get_orders_handler()->add_order_refund( $order_refund, $reason_code );
-		}
```

Comment:
Same comment above about a missing method body.

Reply:
Resolved in 5927703dbae31f24e07d934986af7f999ea92507